content/en/docs/ops/configuration/telemetry/monitoring-multicluster-prometheus/index.md

```diff
@@ -0,0 +1,164 @@
+separate meshes in a single place. Or maybe you have other motivations. Whatever your reason is, you’ll need some special
+configurations to make it all work.
+
+{{< image width="80%"
+    link="./external-production-prometheus.svg"
+    alt="Architecture of external Production Prometheus for monitoring multicluster Istio."
+    caption="External Production Prometheus for monitoring multicluster Istio"
+    >}}
+
+{{< warning >}}
+This guide does not cover securing access to Prometheus. It demonstrates providing connectivity to cluster-local Prometheus
+instances in a few simple ways. For production use cases, it is recommended to secure access to each Prometheus endpoint
+with HTTPS, as well as taking appropriate precautions such as using an internal load-balancer instead of a publicly-accessible
+endpoint and/or properly configuring firewall rules.
+{{< /warning >}}
+
+Istio provides a way to expose cluster services externally via [Gateways](/docs/reference/config/networking/gateway/).
+You can configure an ingress gateway for the cluster-local Prometheus, providing external connectivity to the in-cluster
+Prometheus endpoint.
+
+For each cluster, follow the appropriate instructions from the [Remotely Accessing Telemetry Addons](/docs/tasks/observability/gateways/#option-1-secure-access-https) task. And you
+**SHOULD** establish secure (HTTPS) access.
+
+After that, you will need to configure your external Prometheus instance to access the cluster-local Prometheus instances.
+This can be achieved with a configuration like the following (replacing the gateway address and cluster name):
+
+{{< text yaml >}}
+scrape_configs:
+  - job_name: 'federate-{{CLUSTER_NAME}}'
+    scrape_interval: 15s
+
+    honor_labels: true
+    metrics_path: '/federate'
+
+    params:
+      'match[]':
+        - '{job="pilot"}'
+        - '{job="envoy-stats"}'
+
+    static_configs:
+      - targets:
+        - '{{GATEWAY_IP_ADDR}}:15030'
+        labels:
+          cluster: {{CLUSTER_NAME}}
+{{< /text >}}
+
+Notes:
+
+* `CLUSTER_NAME` should be set to the same value which is used to create the cluster (set via `values.global.multiCluster.clusterName`).
+
+* No authentication to the Prometheus endpoint(s) is provided. This means that anyone can query your
+cluster-local Prometheus instances. This may not be desirable.
+
+* Without proper HTTPS configuration of the gateway, everything is being transported via plaintext. This may not be
+desirable.
+
+### Production Prometheus from one of the clusters
+
+If you desire to run a Prometheus in one of the clusters, establish connectivity from the production instance of
+Prometheus to each of the cluster-local Prometheus instances within the mesh.
+
+This is really just a customization of the process for external federation, which you can achieve by configuring the `Gateway`,
+`VirtualService`, and `DestinationRule` in each remote cluster.
```

Comment:
```suggestion
This is really just a variation of the configuration for external federation. In this case the configuration on the cluster running the production Prometheus is different from the configuration for remote cluster Prometheus scraping.
```

Reply:
I'd prefer the previous description. I think this change is a little bit ambiguous in terms of the remote cluster setup.

Reply:
Sorry if my attempt at clarifying is not correct but the current wording is very unclear. I couldn't really understand what it's trying to say ... hence my best guess was wrong :)

1. `customization of the process` seems like a pretty awkward way to describe this. It's not really a `customization`, and there is no `process`. It's just a slightly different configuration model.
2. what's different about the `Gateway` configuration? You need it for both models.
3. What `VirtualService` and `DestinationRule` are you referring to? There is no mention of them in this doc. Either explain what needs to be configured, or don't mention them.

Reply:
yeah, agree with your above assertions. let me fix it.